examples/diagram-generator/index.html

```diff
@@ -73,23 +71,24 @@
         import { Diagram  } from '../../dist/quick-erd.js';
 
         const input = 
-`departments /insert 2
-   name /nn
-   location
-   country
-   employees /insert 4
-      name /nn vc50
-      email /lower
-      cost center num
-      date hired
-      job vc255
+`
+departments
+    name /nn
+    location
+    country
+employees
+    departments_id /fk departments
+    name /nn vc50
+    email /lower
+    cost center num
+    date hired
+    job vc255
 
 view emp_v departments employees
 
 # settings = { "prefix": null, "semantics": "CHAR", "DV": false }
 `
-        // new quickERD.Diagram( quickSQL.toERD( input ), document.getElementById( 'quickERD' ) );
```

Comment:
Same as above, this comment was there to demonstrate the libraries use with the UMD build

Reply:
I will return the commented line